legacy/skills/src/main/java/cm/aptoide/skills/games/BackgroundGameService.kt

```diff
@@ -69,7 +72,18 @@ class BackgroundGameService : Service(), GameStateListener {
         getString(R.string.playing_game_notification_title),
         getString(R.string.playing_game_notification_body)
       )
-      this.startForeground(NOTIFICATION_SERVICE_ID, notification)
+      try {
+        if (Build.VERSION.SDK_INT > Build.VERSION_CODES.TIRAMISU) {
```

Comment:
It's better to specify the android version where this condition is needed, like this:
```suggestion
        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.UPSIDE_DOWN_CAKE) {
```